src/AppTypes/WordPressType.php

```diff
@@ -71,4 +39,46 @@ public function getFrameworkConfig()
         return $config;
     }
 
+    public static function getArgs()
+    {
+        return [
```

Comment:
Overall this is an improvement, though i'm in love with losing named parameters from a self documentation standpoint, darn you php.

You could use a fluent api to set things, but i'm not sure if thats an improvement or not